### ISSUE TITLE
Use left and right keys to navigate between code samples

### ISSUE THREAD

**Issue by TheEvilDev** [thejameskyle/spectacle-code-slide#5]
Currently it seems like I have to switch key strokes from left and right to up and down when on the code slide. Is there a way to change this so it flows the same as the rest of the presentation?

I also noticed when navigating through the presenter, it seems to load the last code sample first instead of the other way around.

**Comment by joeyrobert** [comment]
+1 for this. Would this also fix the issue where Spectacle code slides show up very poorly when converting to print? Only the first code segment shows up in export and I can't seem to figure out how to print all sections of a code slide.

**Comment by joshburgess** [comment]
Yup, most wireless clickers don't have up and down arrows... +1 for this.

**Comment by joshburgess** [comment]
+1 on loading location blocks out of order too. Trying to fake this <- -> navigation right now with multiple slides to highlight different blocks and they aren't appearing in the order you'd expect them to.

**Comment by atuttle** [comment]
@joshburgess perhaps you are experiencing #17 ?

**Comment by joshburgess** [comment]
I don't believe so. I wasn't using presenter mode.

**Comment by joshburgess** [comment]
Disregard my last comment... I'm not sure what was actually causing that issue where I needed to prepend a fake [0, 0] slide to get the first highlighted code block to show correctly... It stopped editing halfway through creating my slide deck, and I was changing so many different things in a hurry I don't know what fixed it.
